coreneuron/io/global_vars.cpp

```diff
@@ -32,6 +33,16 @@ void hoc_register_var(DoubScal* ds, DoubVec* dv, VoidFunc*) {
     }
 }
 
+// Only checked here in file mode. Already checked via corenrn_units_use_legacy
+// in direct mode since in the latter case we do not want to abort if
+// inconsistent.
+static void chk_nrnunit_consist(int b) {
+    if (b != CORENRN_LegacyFR) {
```

Comment:
Is it possible to use the function above?
And if the function can be a boolean with a better name, I prefer.

Reply:
Changed  ```CORENRN_LegacyFR``` to ```CORENRN_UseLegacyUnits```

Reply:
Sorry, I mean the **argument** can be a boolean and have a better name!

Reply:
Inlined the function